.github/workflows/changelog.yml

```
@@ -0,0 +1,27 @@
+name: Changelog
+
+on:
+  pull_request
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+    cancel-in-progress: true
+
+jobs:
+  danger:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7'
+    - name: Copy Gemfiles to repo root
+      run:
+        cp ./ci/danger/Gemfile ./ci/danger/Gemfile.lock ./
+    - name: Setup Bundler
+      run: ./ci/danger/setup_bundler.sh
```

nit: alternatively can something like this work

```suggestion
    - name: Copy Gemfiles to repo root
      run:
        cp ./ci/danger/Gemfile ./ci/danger/Gemfile.lock ./
    - name: Setup Bundler
      run:  $(cd ./ci/danger && ./ci/danger/setup_bundler.sh)
```